### DISCUSSION TITLE
Kommunikation unter mehren Pybricks Anwendern

### DISCUSSION THREAD

**Discussion by msw-home** [pybricks/support#1757] [Q&A]
Hallo zusammen,
gibt es eine Pybricks Variante, wo mehrere Teilnehmer online auf ein gemeinsames Projekt zugreifen können.
Wo was wie Fernwartung!
z.B. Opa mit Enkel

**Comment by msw-home** [comment]
Sorry, Schreibfehler ohne Ende!
Das ganze ist als Frage gemeint!
Gibt es eine Variante von Pybricks, die wie eine Fernwartung funktioniert?
Ich möchte online mit meinem Enkel, Programm erstellen, also müssen beide auf das gleiche Programm zugreifen können.
Gruß Manfred 😉

**Comment by laurensvalk** [comment]
That would be pretty awesome, but it's not something we expect to be working on any time soon. Thanks for asking!
